packages/node/src/configure/SubqueryProject.ts

```diff
@@ -60,17 +83,28 @@ export class SubqueryProject {
     const manifest = parseStellarProjectManifest(rawManifest);
 
     if (manifest.isV1_0_0) {
-      return loadProjectFromManifest1_0_0(
+      return loadProjectFromManifestBase(
         manifest.asV1_0_0,
         reader,
         path,
-        networkOverrides,
         root,
+        networkOverrides,
       );
     } else {
       NOT_SUPPORT(manifest.specVersion);
     }
   }
+
+  async applyCronTimestamps(
```

Comment:
seems not used?

Reply:
it is called inside node-core
https://github.com/subquery/subql/blob/1e8ff96f78ccaea6382035ff60f3a82a0d2f41ea/packages/node-core/src/indexer/project.service.ts#L83